config/locales/ru.yml

```diff
@@ -1096,7 +1097,8 @@ ru:
     reference: "Ссылка"
     refund: "Возврат"
     refund_amount_must_be_greater_than_zero:
-    refund_reasons: "Причины возврата"
+    refund_reasons: "Причины возврата денег"
+    return_reasons: "Причины возврата товара"
```

Comment:
Ah, excuse me. I didn't saw the difference between `refund_reasons` and `return_reasons`. Could you please put `return_reasons` into correct alphabetical order? Thanks.